appveyor/compile.ps1

```diff
@@ -185,7 +185,8 @@ call configure.bat ^
 	`"--enable-object-out-dir=$objOutDirectory`" ^
 	`"--with-prefix=$outputDirectory`" ^
 	`"--with-php-build=$dependenciesDirectory`" ^
-	--with-mp=auto
+	--with-mp=auto ^
+	--without-xmlrpc
```

Comment:
Thanks! I thought there could be some problems with it... But I wanted to see the AppVeyor running tests :wink: